contrib/runners/action_chain_runner/action_chain_runner.py

```diff
@@ -321,7 +323,7 @@ def run(self, action_parameters):
             }
 
         parent_context = {
-            'execution_id': self.execution_id
+            'execution_id': self.execution_id,
```

Comment:
Wonder if this is a habit from writing Go. :) The compiler makes you do this for literals